drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -761,6 +774,54 @@ function DragAndDropEditBlock(runtime, element, params) {
 
                 },
 
+                adjustDataScaleForOldVersion() {
+                    // It's different from New version. the old version using Percentage between BkgImage & Zones to locate the x,y,width,height of zones.
+                    if (_fn.is_old_version === true) {
```

Comment:
- It's not a page loading issue. It's a compatible issue between new and old component version while calculating locations of background image + zones position when background image size is more larger than cavas.
 - In old version of component , they use percentage between bk ground image and zones.
 - So I added same machenism in our new version.